apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx

```diff
@@ -13,4 +13,47 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
 
 <Callout type="warning" emoji="⚠️">
 This documentation is currently under development.
-</Callout>
\ No newline at end of file
+</Callout>
+
+
+This guide is the first step in developing a **FastStore** website integrated with the **VTEX** Platform. In the following section, you will set up your VTEX environment and install all the necessary apps to start developing with FastStore.
+
+Notice that you must complete the following steps before developing your storefront or configuring your website DNS because needed dependencies, such as [VTEX Intelligent Search](https://help.vtex.com/tracks/vtex-intelligent-search), will be installed and configured throughout this process.
+
+---
+
+## Before you start
+
+Before proceeding any further, make sure you have:
+
+1. Access to a **VTEX** account.
+2. The **VTEX IO CLI** installed on your machine. Please refer to [this guide](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-vtex-io-cli-installation-and-command-reference) for more information.
+
+---
+
+## Step by step
+
+You will use the FastStore plugin for the VTEX IO CLI to install, with a single command, all dependencies needed for developing a FastStore website.
+
+1. Log in to your VTEX account:
+
+<Callout type="info" emoji="ℹ️">
+   Remember to replace the values between curly brackets according to your scenario.
+</Callout>
+
+```
+vtex login {account}
+```
+
+2. Install the FastStore plugin for the VTEX IO CLI:
+
+```
+vtex plugins install faststore
+```
+
+3. Set up your VTEX account for FastStore by running the following command:
+```
+vtex faststore setup
```

Comment:
We intend to discontinue this to prioritize the onboarding to be done through the faststore app directly from the admin. cc @gvc

Reply:
@emersonlaurentino is that already a reality?

Reply:
I'm not sure if this is a reality yet, but I agree that we should move in that direction. I'd leave the step in the readme right now and remove it once we're certain that this is a reality.